src/app/GitUI/CommandsDialogs/RepoHosting/ForkAndCloneForm.cs

```diff
@@ -34,12 +34,14 @@ public partial class ForkAndCloneForm : GitExtensionsForm
         #endregion
 
         private const string UpstreamRemoteName = "upstream";
+        private readonly IGitUICommands _commands;
         private readonly IRepositoryHostPlugin _gitHoster;
         private readonly EventHandler<GitModuleEventArgs>? _gitModuleChanged;
 
-        public ForkAndCloneForm(IRepositoryHostPlugin gitHoster, EventHandler<GitModuleEventArgs>? gitModuleChanged)
+        public ForkAndCloneForm(IGitUICommands commands, IRepositoryHostPlugin gitHoster, EventHandler<GitModuleEventArgs>? gitModuleChanged)
```

Comment:
```suggestion
        public ForkAndCloneForm(IServiceProvider serviceProvider, IRepositoryHostPlugin gitHoster, EventHandler<GitModuleEventArgs>? gitModuleChanged)
```

Reply:
We don't do this for the forms, only for controls. I haven't given this too much thought, but on the surface it's probably a good change to make across the codebase.